packages/select/src/components/FzSelectOption.vue

```diff
@@ -6,6 +6,9 @@
     :title="option.label"
     @click="
       () => {
+        if (option.disabled || option.readonly) {
```

Comment:
I think it is preferable to assign the `disabled` attribute

Reply:
do you mean to have just option.disabled here? I had to add readonly because there were 2 different visuals for those but the same behavior

Reply:
No, I mean that it is preferable to use the disabled attribute to disable the click.
eg:
`:title="option.label"
:disabled="option.disabled || option.readonly"`